app.py

```diff
@@ -24,14 +31,18 @@ def home():
 def index():
     return render_template('form.html')
 
+# output after you fill out /form
 @app.route('/output',  methods=['POST'])
 def success():
     if request.method == 'POST':
         #email = request.form['email']
         #return render_template('success.html', email=email)
+
         address = request.form['address']
         zipcode = request.form['zipcode']
-        zestimate, rent_zestimate, home_size, num_of_bedrooms, num_of_bathrooms, property_size, year_built = zillow_address_pull.zillow_info(address, zipcode)
+
+        zestimate, rent_zestimate, home_size, num_of_bedrooms, num_of_bathrooms, property_size, year_built, last_sold_price, \
```

Comment:
This is what I was talking about btw. You won't have to change the way you unpack this each time you wanna update the function `zillow_address_pull` if you make it return some standard data structure like a dictionary or namedtuple